modules/migration-using-must-gather.adoc

```diff
@@ -73,12 +83,17 @@ endif::[]
 ifdef::oadp-troubleshooting[]
 * Prometheus metrics data dump:
 endif::[]
+.. For OADP 1.2 use the following command:
```

Comment:
A bit of jumping around with OADP 1.2 and 1.3 steps. WHy not group the OADP 1.2 tasks together and then list the 1.3 tasks?

How about lead in with:

Depending on the version of OADP, run the `oc adm must-gather` command for one of the following data collection options:

Reply:
@dfitzmau 
Thank you for your suggestion. Before I updated the PR I discussed the structure with my DPM and our lead writer. We considered the option you suggested. However, since the steps are similar and it is only a minor change in the commands, we decided to use the current structure, which we thought would simplify the information for the reader and make it easier to maintain the documentation in the future.